app/components/WardTile.tsx

```diff
@@ -29,10 +31,8 @@ type WardDTO = {
 
 export type WardTileProps = {
   ward: WardDTO,
-  isSelected: boolean,
-  onTileClicked?: () => void,
   onEditClicked?: () => void
-}
+} & CardProps
```

Comment:
When doing `type ABC = { ... } & D` I usually prefer to have the `D` come first, meaning `type ABC = D & { ... }`, or applied to this specific case:

```ts
export type WardTileProps = CardProps & {
  ward: WardDTO,
  onEditClick?: () => void
}
```